src/app/home/home.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
I think we should manually remove unit tests which are being generated by cli (it will cause tonns of problems in future)